draft-ietf-quic-transport.md

```diff
@@ -4747,9 +4747,10 @@ QUIC acknowledgements are irrevocable.  Once acknowledged, a packet remains
 acknowledged, even if it does not appear in a future ACK frame.  This is unlike
 TCP SACKs ({{?RFC2018}}).
 
-An ACK frame acknowledges packets in the same packet number space as the packet
-in which it is contained.  As such, the same packet number can be acknowledged
-in different packet number spaces.
+A packet number can be used in multiple packet number spaces. An acknowledgement
```

Comment:
This runs afoul of the essential complaint in Dmitri's issue, namely that "packet number" refers to the scoped identifier (and not an unadorned numeric value).  

You might instead say:

```suggestion
The same numeric value might be used to identify packets from multiple packet number spaces. An acknowledgement
```

Reply:
I don't agree. A packet number refers to the numeric value, does it not? I don't think changing it to "numeric value" makes a difference, it seems to be just another way to say the same thing. I do like the second part of your change, so I'll take that: "The same packet number might be used to identify packets from ..."

It seems perfectly reasonably to say that a packet number can be reused. We currently say that it "will be" reused, which is obviously incorrect. That is what I read Dmitri's issues to be.